### PR TITLE
Add NO_SIGNAL_STATE

### PR DESCRIPTION
**Pull request by QuentinPerez**

### CODE REVIEW

**Comment by moul** [comment]
Redundant with https://github.com/scaleway/initrd/blob/master/Linux/tree-common/functions#L96

And you should also add the check here: https://github.com/scaleway/initrd/blob/master/Linux/tree-common/init#L163

**Comment by QuentinPerez** [comment]
thank you :+1:

File: Linux/tree-common/init [hunk 1]
```diff
@@ -224,7 +224,9 @@ log_end_msg
 
 
 # Signal to the console that the server is booted
-signal_state booted
+if [ "$(get_any DONT_SEND_STATE)" != "1" ]; then
```
Comment (moul) [inline Linux/tree-common/init]: can you rename it to `NO_SIGNAL_STATE` ?

**Comment by moul** [comment]
Can you apply the same rule for `signal_state kernel-started` ?

**Pull request closed without merging**